src/main/java/Duke.java

```diff
@@ -1,10 +1,116 @@
+                        } else {
+                            descriptionString = descriptionString + splitInput[i] + " ";
+                        }
+                    }
+                    while (byIndex != wordsCount) {
+                        byString = byString + splitInput[byIndex] + " ";
+                        byIndex++;
+                    }
+                    newItem = new Deadline(descriptionString, byString);
+                    TaskArray[noOfTasks] = newItem;
+                    noOfTasks = noOfTasks + 1;
+                    System.out.println("Got it. I've added this task:");
+                    System.out.println(TaskArray[noOfTasks - 1].toString());
+                    System.out.println("Now you have " + noOfTasks + " tasks in the list.");
+                    System.out.println(sectionDivider);
+                }
+                if (splitInput[0].equals("event")) {
+                    int wordsCount = splitInput.length;
+                    String descriptionString = "";
+                    String atString = "";
+                    int atIndex = 0;
+                    for (int i = 1; i != wordsCount; i++) {
+                        if (splitInput[i].equals("/at")) {
+                            atIndex = i + 1;
+                            break;
+                        } else {
+                            descriptionString = descriptionString + splitInput[i] + " ";
+                        }
+                    }
+                    while (atIndex != wordsCount) {
+                        atString = atString + splitInput[atIndex] + " ";
+                        atIndex++;
+                    }
+                    newItem = new Event(descriptionString, atString);
+                    TaskArray[noOfTasks] = newItem;
+                    noOfTasks = noOfTasks + 1;
+                    System.out.println("Got it. I've added this task:");
+                    System.out.println(TaskArray[noOfTasks - 1].toString());
+                    System.out.println("Now you have " + noOfTasks + " tasks in the list.");
+                    System.out.println(sectionDivider);
+                }
+                if (splitInput[0].equals("todo")) {
+                    int wordsCount = splitInput.length;
+                    String descriptionString = "";
+                    for (int i = 1; i != wordsCount; i++) {
+                        descriptionString = descriptionString + splitInput[i] + " ";
+                    }
+                    newItem = new Todo(descriptionString);
+                    TaskArray[noOfTasks] = newItem;
+                    noOfTasks = noOfTasks + 1;
+                    System.out.println("Got it. I've added this task:");
+                    System.out.println(TaskArray[noOfTasks - 1].toString());
+                    System.out.println("Now you have " + noOfTasks + " tasks in the list.");
+                    System.out.println(sectionDivider);
+                }
```

Comment:
Perhaps you could consider using switch statement to prevent invalid input from the user.